PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/ServerExplorerToolWindowFactory.java

```diff
@@ -114,6 +118,19 @@ public void createToolWindowContent(@NotNull final Project project, @NotNull fin
         final List<? extends com.microsoft.azure.toolkit.intellij.common.component.Tree.TreeNode<?>> modules = AzureExplorer.getModules().stream()
                 .map(m -> new com.microsoft.azure.toolkit.intellij.common.component.Tree.TreeNode<>(m, tree)).collect(Collectors.toList());
         modules.forEach(azureRoot::add);
+        AzureExplorer.explorerExtensionPoint.addExtensionPointListener(new ExtensionPointListener<IExplorerContributor>() {
```

Comment:
please add another listener in `AzureExplorer` to guarantee `AzureExplorer.getModules()` returns all modules.